src/ApiPayloadConverter.js

```diff
@@ -22,6 +22,83 @@ class ApiPayloadConverter {
 
     return payload;
   }
+
+  /**
+   * @description Converts the mail payload for the AWS Simple Mail Service (AWS JavaScript SDK v3).
+   * @param {Object} originalPayload The original payload (provider agnostic).
+   * @returns {Object} The payload according to AWS SDK specification.
+   */
+  static awsSES(originalPayload) {
+
+    // Clone payload
+    const payload = Object.assign({}, originalPayload);
+
+    // Substitute keys
+    if (payload.to) {
+      payload.Destination = {
+        ToAddresses: [payload.to]
+      };
+      delete payload.to;
+    }
+
+    if (payload.from) {
+      payload.Source = [payload.from];
+      delete payload.from;
+    }
+
+    if (payload.replyTo) {
+      payload.ReplyToAddresses = [payload.replyTo];
+      delete payload.replyTo;
+    }
+
+    if (payload.subject) {
+      if (!payload.Message) {
+        payload.Message = {}
+      }
+
+      if (!payload.Message.Subject) {
+        payload.Message.Subject = {}
+      }
+
+      payload.Message.Subject.Data = payload.subject;
+      payload.Message.Subject.Charset = 'UTF-8';
+      delete payload.subject;
+    }
+
+    if (payload.html && payload.html !== '') {
+      if (!payload?.Message) {
+        payload.Message = {}
+      }
+
+      if (!payload.Message.Body) {
+        payload.Message.Body = {}
+      }
+
+      payload.Message.Body.Html = {
+        Charset: 'UTF-8',
+        Data: payload.html
+      };
+      delete payload.html;
+    }
+
+    if (payload.text && payload.text !== '') {
```

Comment:
```suggestion
    if (payload.text) {
```

It's up to the developer to make sure they send only the required data. Are there any cases in which the payload is en empty string?

Reply:
Good point.  I'll remove that check.